docs/source/ui/pages/table.md

```diff
@@ -0,0 +1,42 @@
+## Table
+
+### Overview
+
+The table helps to explore and compare training runs results, logged hyper-parameters, system info, etc. As well as to apply quick actions (hide, delete, archive, etc.) over a batch of selected items within a few clicks.
```

Comment:
The Aim Table helps to explore and compare training runs - their results, logged hyper-parameters, system info, etc. You can also use the table to hide, delete, archive and compare selected training runs in a few clicks.